Updates to use metrics (#87)
chaoss/wg-ai-alignment: metrics/ai-alignment-community-governed-use/ai-use-policy-change.md

# Policy Change

**Question:** How  does a community's AI policy changes over time, and how that compares to shifts happening across other projects.

> This metric is part of the [AI Alignment - Community Governed Use](../ai-alignment-community-governed-use/metric-model-ai-alignment-community-governed-use.md) metrics mode.

Reviewer: ```suggestion
This metric is part of the [AI Alignment - Community Governed Use](../ai-alignment-community-governed-use/metric-model-ai-alignment-community-governed-use.md) metrics model.
```

### Filters

> Filters based on [ai-use-policy-specificity](metrics\ai-alignment-community-governed-use\ai-use-consent-policy-specificity.md)

Reviewer: This link does not work. It returns a 404 error. I am not 100% sure about the fix, but it might be this:

```suggestion
Filters based on [ai-use-policy-specificity](ai-use-consent-policy-specificity.md):
```

> -  Code contributions (PRs, issues, comments)

Reviewer: ```suggestion
- Code contributions (PRs, issues, comments)
```

- Notetaker / meeting bots (recorded discussion, closed or small-group content)
- Content (documentation, blogs, design assets)
- Moderation actions
- Review (who or what may review using AI)
- Autonomous / agentic use
- Environmental Impact (energy use, water use, hardware/carbon footprint)
- Infrastructure strain (server load, hardware cost/financing)
- Data use for training (platform user data)

### Visualizations

> - A timeline that shows how policy language changed over time and whether tha twas to be more or less strict. 

Reviewer: ```suggestion
- A timeline that shows how policy language changed over time and whether that was to be more or less strict.
```
